app/controllers/dashboard_controller.rb

```diff
@@ -47,8 +47,8 @@ def index
 
   def collections_list
     collections = Collection.includes(:owner).distinct
-    @document_sets = DocumentSet.includes(:owner).distinct
-    @collections = (collections + @document_sets).sort { |a,b| a.title <=> b.title }
+    document_sets = DocumentSet.includes(:owner).distinct
+    @collections_and_document_sets = (collections + document_sets).sort { |a,b| a.title <=> b.title }
```

Comment:
I renamed this variable to make it more clear what kind of data it is housing.

Reply:
Hooray!